static.config.js

```diff
@@ -367,27 +367,6 @@ const makeDepartmentPages = async (client, langCode) => {
     if (!infoPage.department) continue;
 
     infoPage.type = 'info';
-
-    let matchingDepartmentIndex = departments.findIndex(
-      d => d.id === infoPage.department.id,
-    );
-
-    if (departments[matchingDepartmentIndex]) {
-      // Only add the related link if it's not a top service
-      if (
-        !departments[matchingDepartmentIndex].topServiceIds.includes(
-          infoPage.id,
-        )
-      ) {
-        departments[matchingDepartmentIndex].relatedLinks.push(infoPage);
-      }
-
-      // Update the department on the page
-      const departmentCopy = JSON.parse(
-        JSON.stringify(departments[matchingDepartmentIndex]),
-      );
-      infoPage.department = departmentCopy;
-    }
```

Comment:
Since we're now creating our own services and links, we don't need to run these related conditions!